native/src/components/Pagination.tsx

```diff
@@ -29,11 +32,20 @@ type PaginationProps = {
 
 const Pagination = ({ slideCount, currentSlide, goToSlide }: PaginationProps): ReactElement => {
   const goToSlideIndex = (index: number) => () => goToSlide(index)
+  const { t } = useTranslation()
```

Comment:
since we are using not different namespaces in this component i would prefer using here directly and remove the `ns: 'error'`
```suggestion
  const { t } = useTranslation('error')
```